src/scene/getExportCanvasPreview.ts

```diff
@@ -53,3 +53,45 @@ export function getExportCanvasPreview(
   );
   return tempCanvas;
 }
+
+export function getExportSvgPreview(
```

Comment:
I'd rename this and remove the `preview` since it no longer has anything to do with preview. Also applies to the function above, and to the filename in general. So maybe leave this as last step or another PR, to keep the diffs readable.

Reply:
Yes, I didn't like the name. I was just copying the style that was already in the file.